scripts/build/libc/glibc.sh

```diff
@@ -170,6 +170,9 @@ do_libc_backend() {
                              libc_target="${target}"
 
         CT_Popd
+        if [ "${multi_dir}" != "." ]; then
+            CT_EndStep
```

Comment:
@bhundven : you could probably fold that directly in the corresponding commit
without the need for @stilor to rebase-and-repush, no?